Source/monster.cpp

```diff
@@ -371,7 +371,7 @@ void PlaceGroup(int mtype, int num, UniqueMonsterPack uniqueMonsterPack, int lea
 
 				if (minion._mAi != AI_GARG) {
 					minion.ChangeAnimationData(MonsterGraphic::Stand);
-					minion.AnimInfo.CurrentFrame = GenerateRnd(minion.AnimInfo.NumberOfFrames - 1);
+					minion.AnimInfo.setCurrentFrame(GenerateRnd(minion.AnimInfo.NumberOfFrames - 1));

```

Comment:
I am wondering if it's not a bug.
Shouldn't this be `GenerateRnd(minion.AnimInfo.NumberOfFrames)` ?